elastalert/alerters/ses.py

```diff
@@ -11,8 +11,8 @@ class SesAlerter(Alerter):
     def __init__(self, *args):
         super(SesAlerter, self).__init__(*args)
 
-        self.aws_access_key_id = self.rule.get('ses_aws_access_key_id')
-        self.aws_secret_access_key = self.rule.get('ses_aws_secret_access_key')
+        self.aws_access_key_id = self.rule['ses_aws_access_key_id']
```

Comment:
I don't think these are required. If the user chooses to rely on the ses_aws_profile then there should not be a need to also specify the key and secret separately.

Reply:
The body of the pull request is incorrect. I'm sorry. Only ``sns_topic_arn`` is required before modification. The contents of the correction are the correction, the addition of the test code, and the correction of the document in the writing method when the initial value is not specified.

Reply:
My understanding is that `foo['bar']` will through an exception if the key doesn't exist, while `foo.get('bar')` will return None if the key doesn't exist. Since your change removed the `.get` syntax won't that effectively break these alerters for anyone that uses the `ses_aws_profile` parameter?

Reply:
Revert aws_access_key_id and aws_secret_access_key.